pmoves/services/gateway-agent/README.md

```
@@ -0,0 +1,268 @@
+
+```bash
+# Install dependencies
+pip install -r requirements.txt
+
+# Run with uvicorn
+uvicorn app:app --host 0.0.0.0 --port 8100 --reload
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)